include/ofi_mem.h

```diff
@@ -258,6 +261,7 @@ struct util_buf_attr {
 	util_buf_region_alloc_hndlr 	alloc_hndlr;
 	util_buf_region_free_hndlr 	free_hndlr;
 	void 				*ctx;
+	uint8_t				track_index;
```

Comment:
As long as we have the footer available, storing the index adds slighty more work when new buffers are allocated.  Maybe we should drop track_index and always make it available when using footers.  It might even be useful for debugging.